source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+.. _pdps-group-replication:
+
+********************************************************************************
+High Availability Solution with Group Replication. Technical Overview
+********************************************************************************
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+=================
+
+Every architecture and deployment depends on customer requirements and application demands for high availability and the estimated level of usage.  For example, using a high read or a high write application, or both with  99.999% availability.
+ 
+This document gives architecture and deployment recommendations along with a technical overview for a solution that provides a high level of high availability and assumes the usage of high read / write applications (20K or more queries per second). For step-by-step deployment guidelines, refer to :ref:`deploy`.
+
+This solution assumes the use of |PS| based deployment variant of Percona Distribution for MySQL with `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
+
+.. _layout:
+
+Architecture layout
+=====================
+
+.. image:: _static/images/group_replication_ha.png
+
+Components
+----------------
+
+The architecture is composed of two main layers:
+
+- :ref:`Connection and distribution layer <connection>`
+- :ref:`Relational Database Management System (RDBMS) layer <rdbms>`
+
+.. _connection:
+
+Connection and distribution layer 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connection layer consists of the following:
+
+- **Application to proxy redirection mechanism.** It can be anything from a Virtual IP managed by Keepalived local service to a DNS resolution service like Amazon Route 53. The mechanism's function is to redirect the traffic to the active Proxy node.
+- **Proxy connection distribution.** It is composed of two or more nodes. Its role is to redirect the traffic to the active nodes of the Group Replication cluster. In cases like ProxySQL where the proxy is a level 7 proxy and can perform a read / write split, this layer is also in charge of redirecting writes to the Primary node and reads to the replicas, and of high availability to prevent a single point of failure.
```

Comment:
The repetition of "It" and "its" is vague.
Maybe
The distribution is composed of two or more nodes and redirects the traffic to Group Replication cluster active nodes.

Reply:
fixed